src/coreclr/inc/readytorun.h

```diff
@@ -20,7 +20,7 @@
 // If you update this, ensure you run `git grep MINIMUM_READYTORUN_MAJOR_VERSION`
 // and handle pending work.
 #define READYTORUN_MAJOR_VERSION 12
-#define READYTORUN_MINOR_VERSION 0x0000
+#define READYTORUN_MINOR_VERSION 0x0001
```

Comment:
@jkotas does this require a major version bump, or does it not matter since we're still in the early previews of net10?

Reply:
Minor version bump is fine for this case. It does not matter a whole lot since we took a major version bump in .NET 10 and we are likely going to take another one before .NET 10 ships.